cnames_active.js

```diff
@@ -3245,6 +3245,7 @@ var cnames_active = {
   "temme": "shinima.github.io/temme",
   "template-pro": "template-pro.github.io",
   "templates.linksharer": "linksharer.github.io/Templates", // noCF
+  "tempmail": "email-echo-replica-project.vercel.app"
```

Comment:
```suggestion
  "tempmail": "cname.vercel-dns.com", // noCF
```
As the CI notes, when using Vercel, you'll want to target `cname.vercel-dns.com` with the Cloudflare proxy disabled.